tests/test_token_blacklist.py

```diff
@@ -190,3 +191,32 @@ def test_jti_field_should_contain_uuid_hex_strings(self):
         actual_hexes = [i.jti_hex for i in OutstandingToken.objects.all()]
 
         self.assertEqual(actual_hexes, self.expected_hexes)
+
+
+class TokenVerifySerializerShouldHonourBlacklist(MigrationTestCase):
+    migrate_from = ('token_blacklist', '0002_outstandingtoken_jti_hex')
```

Comment:
this is testing migration but the PR is not showing any migration changes, or it was already done but not tested?

Reply:
Yea those migrations were already done and come from https://github.com/jazzband/django-rest-framework-simplejwt/tree/master/rest_framework_simplejwt/token_blacklist/migrations